### PR TITLE
application lifecycle edit

### PR DESCRIPTION
**Pull request by tsonevn**

### CODE REVIEW

File: docs/core-concepts/application-lifecycle.md [hunk 1]
```diff
@@ -449,8 +449,9 @@ class MyDelegate extends UIResponder implements UIApplicationDelegate {
     }
 }
 ios.delegate = MyDelegate;
-applicationStart();
+applicationStart({ moduleName: "main-page" });
```
Comment (tjvantoll) [inline docs/core-concepts/application-lifecycle.md:452]: Good catch on these 😄

File: docs/core-concepts/application-lifecycle.md [hunk 2]
```diff
@@ -449,8 +449,9 @@ class MyDelegate extends UIResponder implements UIApplicationDelegate {
...
 ```
+> **Note:** While extending `UIApplicationDelegate` in NativeScript TypeScript project we need also to add [tns-platform-declarations](https://www.npmjs.com/package/tns-platform-declarations) plugin as a dependency.
```
Comment (tjvantoll) [inline docs/core-concepts/application-lifecycle.md]: Suggested rewording

> **NOTE**: If you’re using TypeScript in your NativeScript apps, you need to install the [tns-platform-declarations plugin](https://www.npmjs.com/package/tns-platform-declarations) to add typings for native iOS APIs such as `UIApplicationDelegate`.

**Review by tjvantoll — APPROVED** [review]
Looks good! I had a minor rewording suggestion on the  note you added. Let me know what you think about that, and I can go ahead and merge this in.

File: docs/core-concepts/application-lifecycle.md (hunk 2 above)
Comment (tsonevn) [reply docs/core-concepts/application-lifecycle.md]: @tjvantoll, thanks for the suggestion.

File: docs/hardware/location.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ import { isEnabled, enableLocationRequest, getCurrentLocation, watchLocation, di
 
 ## Getting information about a location service
 
-NativeScript has a universal way to check if location services are turned on&mdash;the `isEnabled` method. The method returns a Boolean value (true if the location service is enabled).
+NativeScript has a universal way to check if location services are turned on&mdash;the `isEnabled` method. The method returns a `Promise<boolean>` (resolves `true` or `false` based on the location services availability.).
```
Comment (tjvantoll) [inline docs/hardware/location.md:37]: Given this, it looks like the code samples that come after this sentence are out of date, as they expect `isEnabled()` to return a `boolean`, not a `Promise<boolean>`.

**Comment by tjvantoll** [comment]
Merged with 32975b5fc906235959333c3c34d75eef5417ba8d and 1472ce516d6da45a2a6e2993d8fb07879b191a4b. Thanks @tsonevn!

**Pull request closed without merging**